x/posts/client/cli/tx.go

```diff
@@ -43,9 +44,17 @@ func GetTxCmd(_ string, cdc *codec.Codec) *cobra.Command {
 // GetCmdCreatePost is the CLI command for creating a post
 func GetCmdCreatePost(cdc *codec.Codec) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "create [subspace] [message] [allows-comments]",
+		Use:   "create [subspace] [message] [allows-comments] [[[uri],[provider],[mime-type]]...]",
 		Short: "Create a new post",
-		Args:  cobra.ExactArgs(3),
+		Long: fmt.Sprintf(`
```

Comment:
I think we can improve the command description making it more clear as follows. 

>>>
Create a new post, specifying the subspace, message and whether or not it will allow for comments. 

Optional media attachments are also supported. 
If you with to add one or more media attachment, you have to specify a uri, a provider and a mime type for each. You can do so by concatenating them together separated by a comma (,). 

Example usages: 

%s tx posts create "desmos" "Hello world!" true
%s tx posts create "demos" "A post with media" true "https://example.com,text/plain"
%s tx posts create "desmos" "A post with multiple medias" false "https://example.com/media1,text/plain" "https://example.com/media2,application/json"
>>>